src/applications/auth/containers/AuthApp.jsx

```diff
@@ -250,6 +250,27 @@ export class AuthApp extends React.Component {
         );
         break;
 
+      // Session expired error
+      case '005':
```

Comment:
This might be okay if they're only used here and this case is just following the pattern, but I like to see these values moved into constants so that we don't need to write a comment to add meaning to it (`case SESSION_EXPIRED_ERROR:`) or something